File: Leetcode/PHP/1. Two Sum.php
<?php

class Solution {
    public function twoSum($nums, $target) {
        for ($i = 0; $i < count($nums); $i++) {
            for ($j = $i + 1; $j < count($nums); $j++) {
                if ($nums[$i] + $nums[$j] == $target) {
                    return [$i, $j];
                }
            }
        }
        return [];
    }
}

$sol = new Solution();
$nums = [2, 7, 11, 15];
$target = 9;
$result = $sol->twoSum($nums, $target);
echo "Indices: [" . $result[0] . ", " . $result[1] . "]\n";

?>

File: Leetcode/C++/1_TwoSum.cpp
#include <bits/stdc++.h>
using namespace std;
class Solution {
public:
    vector<int> twoSum(vector<int> nums, int target) {
        for (int i = 0; i < nums.size(); i++) {
            for(int j=i+1; j<nums.size(); j++){
                if(nums[i]+nums[j]==target){
                    return {i, j};
                }
            }
        }
        return {};
    }
};

int main() {
    Solution sol;
    vector<int> nums = {2, 7, 11, 15};
    int target = 9;
    vector<int> result = sol.twoSum(nums, target);
    cout << "Indices: [" << result[0] << ", " << result[1] << "]" << endl;
    
    return 0;
}